operator/v1/types_csi_cluster_driver.go

```diff
@@ -248,6 +254,16 @@ type GCPCSIDriverConfigSpec struct {
 	KMSKey *GCPKMSKeyReference `json:"kmsKey,omitempty"`
 }
 
+// IBMCloudCSIDriverConfigSpec defines the properties that can be configured for the IBM Cloud CSI driver.
+type IBMCloudCSIDriverConfigSpec struct {
+	// encryptionKeyCRN is the IBM Cloud CRN of the customer-managed root key to use
+	// for disk encryption of volumes for the default storage classes.
+	// +kubebuilder:validation:MaxLength:=154
+	// +kubebuilder:validation:MinLength:=144
+	// +optional
+	EncryptionKeyCRN string `json:"encryptionKeyCRN,omitempty"`
```

Comment:
I'm guessing given the precise length, that there's a pattern for this CRN? What is it? What characters are allowed? Can we add a regex?

Reply:
Yes, the IBM CRN object is defined here (sorry, I didn't see any docs with more helpful details)
https://github.com/IBM-Cloud/bluemix-go/blob/master/crn/crn.go

I can add back the regex I had previously, in additional to the min/max length. That was the best interpretation I had into the format for a CRN for an IBM Cloud Key Protect or Hyper Protect root key.

Reply:
Added the regex I had back and made the value required.